malware-family/plugx/match-known-module.yml

```diff
@@ -0,0 +1,63 @@
+rule:
+  meta:
+    name: match known module
```

Comment:
```suggestion
    name: match known PlugX module
```

Reply:
I think the name should include the malware family or at least be a little bit more descriptive.

Reply:
Added a `description` meta field along with the suggested actions above.